$loggingQueries defaults to true for --pretend flag to output sql queries

knvpk opened the pull request:
As in L4.2 $loggingQueries is defaults to true , so when we run "php artisan migrate --pretend" we have sql quiries logged to console. But in L5.0 $loggingQueries defaults to false, so pretend flag not working correctly. May i know the reason why in L5.0 has $loggingQueries =false; if there is any reason to do that then enable query log $db->enableQuerylog in Illuminate\Database\Connection@pretend to work pretnd flag perfectly. (I also commented the enable query log).

JosephSilber committed:
Use model@hydrate in builder@getModels

MalikinSergey committed:
Add throwing exception, when controller action not found

If undefined action passed In UrlGenerator::action(), it's throw fatal exception

`Call to a member function domain() on a non-object`

with fully uninformative debug stack

because route, which was returned by `$this->routes->getByAction($action)` is not checked, before it passes to `$this->toRoute(...` and can be `null` instead of `Route`

So, it seems logical to add some check, as in UrlGenerator::route()

MalikinSergey committed:
fix cs

taylorotwell committed:
Revert dotenv change.

taylorotwell committed:
Add ability to register callback after any bootstrapped.

taylorotwell committed:
Return.

knvpk committed:
defaults t true for --pretend flag to output sql queries

knvpk committed:
Enableing the querylog to work --pretend flag correctly

Removing phpstorm project file from git ignore

Fixed Connection.php

Aligned correctly.

Removed Traling spaces

 defaults t true for --pretend flag to output sql queries

Conflicts:
	src/Illuminate/Database/Connection.php

knvpk committed:
Enableing the querylog to work --pretend flag correctly

Removing phpstorm project file from git ignore

Fixed Connection.php

Aligned correctly.

Removed Traling spaces

GrahamCampbell commented on .gitignore:
```diff
@@ -3,3 +3,4 @@ composer.phar
 composer.lock
 .DS_Store
 Thumbs.db
+.idea
```
Remove this please.

GrahamCampbell commented on src/Illuminate/Database/Connection.php:
```diff
@@ -532,6 +532,8 @@ public function transactionLevel()
 	public function pretend(Closure $callback)
 	{
+
+//        $this->enableQueryLog();
```
Fix cs please

GrahamCampbell commented:
This was surely disabled by default on purpose?

pavankumarkatakam commented:
Can i keep the $loggingQueries to false.

Commit:
Removing phpstorm project file from git ignore

Commit:
Fixed Connection.php

GrahamCampbell commented on src/Illuminate/Database/Connection.php:
```diff
@@ -532,8 +532,8 @@ public function transactionLevel()
 	{
+        	$this->enableQueryLog();
 		$this->pretending = true;
-
```
Please this line please.

GrahamCampbell replied:
Add one after $this->enableQueryLog(); too plz.

Commit:
Aligned correctly.

pavankumarkatakam commented:
I think its clear now :-)

GrahamCampbell commented on src/Illuminate/Database/Connection.php:
```diff
@@ -532,8 +532,10 @@ public function transactionLevel()
+        	
 		$this->pretending = true;
-
+		
```
No trailing whilespace please.

GrahamCampbell commented on src/Illuminate/Database/Connection.php:
```diff
@@ -532,8 +532,10 @@ public function transactionLevel()
 	public function pretend(Closure $callback)
 	{
+        	$this->enableQueryLog();
+        	
```
No trailing whilespace please.

GrahamCampbell commented:
Fix the trailing spaces, then squash to one commit please.

Commit:
Removed Traling spaces

knvpk committed:
Merge branch '5.0' of https://github.com/pavankumarkatakam/framework into 5.0

knvpk committed:
Update versions for 5.1

Conflicts:
	src/Illuminate/Bus/composer.json
	src/Illuminate/Config/composer.json
	src/Illuminate/Database/composer.json
	src/Illuminate/Foundation/Application.php

knvpk committed:
Merge branch '5.0' of https://github.com/pavankumarkatakam/framework into HEAD

Conflicts:
	src/Illuminate/Database/Connection.php

pavankumarkatakam commented:
I dont know whats going on... Is every thing ok now ? @GrahamCampbell

GrahamCampbell commented:
You seem to have rebased against the master instead of 5.0. This can't be merged as is.

pavankumarkatakam commented:
What I'm supposed to do now?

pavankumarkatakam commented:
I have opened with another [pull request] (https://github.com/laravel/framework/pull/7477) with properly commited. please check that  also.
